uniffi_bindgen/src/bindings/python/templates/RecordTemplate.py

```diff
@@ -1,5 +1,12 @@
 class {{ rec.name()|class_name_py }}(object):
-    def __init__(self,{% for field in rec.fields() %}{{ field.name()|var_name_py }}{% if loop.last %}{% else %}, {% endif %}{% endfor %}):
+    def __init__(self, {% for field in rec.fields() -%}
+        {{ field.name()|var_name_py -}}
+                {%- match field.default_value() %}
+            {%- when Some with(literal) %} = {{ literal|literal_py }}
+            {%- else %}
+        {%- endmatch -%}
```

Comment:
FYI, this will cause problems in python if there are non-default fields following a default field:

```
>>> def test(one, two=2, three):
...   pass
  File "<stdin>", line 1
SyntaxError: non-default argument follows default argument>>>                                                       
```

Reply:
Yuck. Ok. This was meant to be a quick PR; I think I favour pulling out defaults until we have the bandwidth for #300.